go/util/clienttest/client_test_suite.go

```diff
@@ -43,6 +43,9 @@ func (suite *ClientTestSuite) SetupSuite() {
 	suite.out = stdOutput
 	suite.err = errOutput
 	exit.Exit = mockExit
+
+	os.Mkdir(suite.DBDir, 0777)
+	os.Mkdir(path.Join(suite.TempDir, "db2"), 0777)
```

Comment:
What's this second directory for? Is it that a number of the tests make a directory called "db2" and then try to use it? If so, that's kind of an unstated dependency...the test code only works because it "knows" that the harness creates this directory. If many of the tests depend on this directory existing in order to be correct, then the path to this dir should be exposed via some member on ClientTestSuite, and the tests should use that -- as we did with `suite.DBDir`